server.js

```diff
@@ -0,0 +1,75 @@
+const express = require('express');
+const app = express();
+const db = require('./db')
+const { User, Product } = db.models;
+const path = require('path');
+
+app.use(require('express-session')({
+  secret: process.env.SECRET,
+  saveUninitialized: false,
+  resave: false,
+}));
+
+const port = process.env.PORT || 3000;
+
+app.use(express.json())
+app.use('/dist', express.static(path.join(__dirname, './dist')));
+
+app.get('/', (req, res, next)=> {
+  res.sendFile(path.join(__dirname, './index.html'));
+});
+
+//GET Route
+app.get('/api/users', (req, res, next)=>{
+  User.findAll()
+    .then(user => res.send(user))
+    .catch(next)
+})
+app.get('/api/products', (req, res, next)=> {
+  Product.findAll()
+    .then(product => res.send(product))
+    .catch(next)
+})
+
+// Create a user
+app.post('/api/users', (req, res, next)=>{
+  User.create(req.body)
```

Comment:
Don't trust `req.body`, if you add an `admin` field onto the user, this means anybody could create an admin user by just attaching `admin: true` to the body. Make sure to filter down the fields in a request to only the fields you accept.